psi-java/src/test/resources/ClassWithNested.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023, TeamDev. All rights reserved.
```

Comment:
This license statement looks a bit shortened.

I suggest we cut it out altogether. Or if we want to emulate the real-world case, use it in full.